### ISSUE TITLE
[REQUEST - LINUX] Compiling the account manager into a Debian archive

### ISSUE THREAD

**Issue by Distill-pixel** [thenewboston-developers/Account-Manager#537]
The APT repository team had attempted to compile the project into a Debian file, we managed to get the account manager to start but it still wouldn't work due to webpack throwing an error that states it wasn't able to compile a module (the module wasn't specified).

Initially, we had problems with index.js but we just had to use webpack which we forgot to use at first and we had to reconfigure the webpack config a bit, but alas, we got it to work. Webpack unfortunately still did fail to build a module apparently which wasn't known until we actually ran the application via the desktop shortcut.

TL;DR: The APT repo team requires any developer to compile the entire account manager into a Debian archive that is installable and give us the instructions so we can do it in the future too.

The shortcut should be the app name in package.json and the general _name_ should be tnb-acm which we can use to start the account manager via the terminal. Also, for the icon, please do use the TNB red logo. 

What would be better is, a deb archive being attached along with the appimage in releases which would make the whole process a lot easier,.
